scripts/data_visualization_p300.py

```diff
@@ -142,7 +142,7 @@ def epo_summary(epos):
     return summary, info_str
 
 
-FIGURES_PATH = Path("/home/jan/bci_data/figures/moabb_erps")
+FIGURES_PATH = Path.home() / "moabb_figures" / "erps"
```

Comment:
```suggestion
FIGURES_PATH = osp.join(Path.home(), "moabb_figures", "erps")
```

Reply:
Does my code not work for you? Or is using osp a better practice?

Reply:
This is better practice, to ensure that your code is OS independant.

Reply:
That should already be the case, see e.g. here: https://docs.python.org/3/library/pathlib.html#pathlib.Path

And the slash operator is the default way to create os independent directories: https://docs.python.org/3/library/pathlib.html#operators

Anyways, if you prefer osp.join of strings in this codebase, I can change the line to:
```python
FIGURES_PATH = osp.join(osp.expanduser("~"), "moabb_figures", "erps")
```
To stay consistently within one path-manipulation class, i.e. `os.path` instead of `pathlib`

Reply:
Yes, I think it is better to stick with os.path for consistency reason for this PR.

I think we could open an issue to switch to `pathlib`! This library is indeed better for handling files and path.